modules/aleph/includes/aleph.user_consent.inc

```diff
@@ -28,59 +28,43 @@ function aleph_user_consent_info() {
 /**
  * Update the user consent.
  *
- * @param object $account
- *   User account.
- * @param string $consent
- *   Consent category.
- *
  * @return bool
  *   TRUE or FALSE depending on the success of the request.
  */
-function aleph_user_consent_add($account, $consent) {
+function aleph_user_consent_add() {
   // Because no request is done, we return FALSE.
   return FALSE;
 }
 
 /**
  * Delete the users consent.
  *
- * @param object $account
- *   User account.
- * @param string $consent
- *   Consent category.
- *
  * @return bool
  *   TRUE or FALSE depending on the success of the request
  */
-function aleph_user_consent_remove($account, $consent) {
+function aleph_user_consent_remove() {
   // There's no user consents, so return FALSE.
   return FALSE;
 }
 
 /**
  * Retrieve any consent record which the user has saved.
  *
- * @param object $account
- *   The user account.
- *
  * @return array
  *   An array of consent arrays sorted by the date they were accepted.
  */
-function aleph_user_consent_get_consent($account) {
```

Comment:
You're right. PHPStorm was bitching about it, so I removed it without thinking. :)